react-components/src/architecture/base/commands/BaseTool.ts

```diff
@@ -150,7 +149,7 @@ export abstract class BaseTool extends RenderTargetCommand {
       if (!(customObject instanceof ThreeView)) {
         return false;
       }
-      return domainObjectPredicate(customObject.domainObject);
+      return domainObjectPredicate(customObject.domainObject as DomainObject);
```

Comment:
?? I get a lint error on this. Why this cast?

Reply:
You get a Lint error? Weird. It's because Typescript is unable to understand that `domainObject` is always of the generic type DomainObjectType, which always extends DomainObject, and thus is always a DomainObject. So, annoyingly, I had to add this cast